tests/emukit/core/test_user_function.py

```diff
@@ -17,13 +17,12 @@ def test_user_function_wrapper_evaluation_no_cost():
     for i, record in enumerate(output):
         assert_array_equal(output[i].X, function_input[i])
         assert_array_equal(output[i].Y, function(function_input[i]))
-        assert output[i].cost is None
 
 
 def test_user_function_wrapper_evaluation_with_cost():
```

Comment:
The tests including 'cost' seem to be very specific now. In a way they just test the extra outputs. But I don't mind very much to leave it like that.

Reply:
Agreed the "no_cost" is a bit weird, changed to "single_output" to reflect extra arguments don't have to be cost